libcloud/compute/providers.py

```diff
@@ -173,6 +173,8 @@
     ('libcloud.compute.drivers.ciscoccs', 'CiscoCCSNodeDriver'),
     Provider.NTTA:
     ('libcloud.compute.drivers.ntta', 'NTTAmericaNodeDriver'),
+    Provider.ECS:
```

Comment:
Is it possible to use some other name for the provider constant? Maybe `ALIYUN_ECS`?

If we call it just `ECS` it's slightly confusing and many users will probably mistaken it for Amazon Elastic Container Service (ECS).